server/auth/auth.go

```diff
@@ -626,11 +626,26 @@ func (auth *AuthManager) RequestWithTimeout(user account.AccountID, msg *msgjson
 }
 
 // Penalize signals that a user has broken a rule of community conduct, and that
-// their account should be penalized.
+// their account should be penalized. It logs error and is used by internal
+// callers.
 func (auth *AuthManager) Penalize(user account.AccountID, rule account.Rule) {
+	if err := auth.penalize(user, rule); err != nil {
+		log.Error(err)
+	}
+}
+
+// Ban signals that a user has broken a rule of community conduct, and that
+// their account should be penalized. It returns an error and is used by remote
+// callers.
+func (auth *AuthManager) Ban(user account.AccountID, rule account.Rule) error {
+	return auth.penalize(user, rule)
+}
```

Comment:
I just needed the error to return to the remote caller, so I broke Penalize up. Penalize is still exactly the same but with an extra debug log for successes.

Reply:
I don't want two nearly identical functions.  Let's just have Penalize return and error and update the interfaces.  Callers can ignore the error if they want.